test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -215,12 +215,12 @@ public void awaitAndCheck() throws InterruptedException, InvocationTargetExcepti
     }
 
     /**
-     * Dispose all the frame(s) i,e both the test instruction frame as
-     * well as the frame that is added via addTestFrame(Frame frame)
+     * Dispose all the windows(s) i,e both the test instruction frame and
+     * the testWindow that is added via addTestWindow(Window testWindow)
```

Comment:
Shouldn't it be `windows` instead of `testWindow`, because we can add plenty of them with `addTestWindow()`?

Otherwise looks good.